pytorch_lightning/loggers/wandb.py

```diff
@@ -293,7 +292,7 @@ def __init__(
         self._wandb_init = dict(
             name=name,
             project=project,
-            id=version or id,
+            id=id,
```

Comment:
I am not following how this would fix the issue. if both version and id are None, we still have this issue right?

Reply:
So whenever the run object is initialized, if id is passed as None, wandb creates it's own id which is used everywhere else.